protocol/cosi.go

```diff
@@ -74,6 +75,9 @@ type CommitmentHook func(in []abstract.Point) error
 // challenge is received
 type ChallengeHook func(ch abstract.Scalar) error
 
+// ResponseHook allows registering a handler when the response is calculated
```

Comment:
will change the comment then, as I only need the children's response - the total response is given with `SignatureHandler`.